astroplan/plots/plots.py

```diff
@@ -0,0 +1,216 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import matplotlib.pyplot as plt
+import numpy as np
+import astropy.units as u
+import random
+
+
+def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
+                 dark_plot=False):
+    """
+    TODO:
+        1) Timezones?
+        2) Limit airmass <= 3.
+        3) Plot lines instead of points.
+        4) Legend should not be transparent.
+        5) dark_plot option
+
+    Makes an airmass vs. time plot.
```

Comment:
Nitpicky, but a general rule: it's good to be *a little* more verbose here, e.g., "Plot the airmass as a function of time for a given target."